playbooks/roles/tarsnap/tasks/main.yml

```diff
@@ -1,11 +1,10 @@
 ---
 
 - include: install.yml
-  tags:
-    - tarsnap
+  tags: tarsnap
+
 - include: configure.yml
-  tags:
-    - tarsnap
   tarsnap_cache: "{{ TARSNAP_CACHE }}"
   tarsnap_keyfile: "{{ TARSNAP_KEY_REMOTE_LOCATION }}"
-  when: "TARSNAP_KEY is not defined or TARSNAP_KEY == ''"
+  when: "not (TARSNAP_KEY is not defined or TARSNAP_KEY == '')"
```

Comment:
@UmanShahzad, wouldn't it be better to drop the negation and use `TARSNAP_KEY is defined and TARSNAP_KEY != ''`. Makes it easier to read and understand.

Reply:
Yep definitely better.